bluebottle/time_based/models.py

```diff
@@ -375,7 +375,6 @@ class DateActivitySlot(ActivitySlot):
     @property
     def required_fields(self):
         fields = [
-            'is_online',
             'start',
```

Comment:
This field should be required on the model. You need to make a choice here before the activity can go live.

Reply:
Nope. This is only needed on slot level. We should remove this all the way later on.